src/components/GordonDialogBox/index.js

```diff
@@ -65,14 +64,14 @@ export default class GordonDialogBox extends Component {
           <DialogContentText>{this.props.text}</DialogContentText>
         </DialogContent>
         <DialogActions>
-          {this.props.cancelButtonName && (
+          {this.props.cancelButtonClicked !== undefined && (
             <Button variant="contained" onClick={this.props.cancelButtonClicked} color="primary">
-              {this.props.cancelButtonName}
+              {this.props.cancelButtonName ?? 'Cancel'}
             </Button>
           )}
-          {this.props.buttonName && (
+          {this.props.buttonClicked !== undefined && (
             <Button variant="contained" onClick={this.props.buttonClicked} color="primary">
-              {this.props.buttonName}
+              {this.props.buttonName ?? 'Okay'}
```

Comment:
Literally every other page that uses this component currently sets the button name to 'Okay', so it was an intuitive default choice